src/OrchardCore/OrchardCore.Apis.GraphQL.Abstractions/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,10 @@
+namespace OrchardCore.Apis.GraphQL
+{
+    public static class StringExtensions
+    {
+        public static string ToGraphQLStringFormat(this string value)
```

Comment:
Looks like we could remove this and use GraphQL.StringExtensions

public static string ToCamelCase(this string s)
and
public static string ToPascalCase(this string s)